src/app/router/Options/Options.tsx

```diff
@@ -55,7 +56,17 @@ function Options() {
             <Route path="confirmPayment" element={<ConfirmPayment />} />
             <Route path="keysend" element={<Keysend />} />
             <Route path="receive" element={<Receive />} />
-            <Route path="transactions" element={<Transactions />} />
+            <Route path="wallet" element={<Wallet />} />
+            <Route path="transactions">
```

Comment:
@reneaaron I tried to redirect the page to the first child in case of a user typing in the "/transactions" url manually to prevent showing a white page. I did not succeed. Maybe you have an idea.

Reply:
I don't think we need to do that.